superset/assets/src/components/Menu/Menu.jsx

```diff
@@ -24,6 +24,7 @@ import MenuObject from './MenuObject';
 import NewMenu from './NewMenu';
 import UserMenu from './UserMenu';
 import LanguagePicker from './LanguagePicker';
+import "./Menu.less"
```

Comment:
linter is complaining about double quotes and lack of `;`

Reply:
```suggestion
import './Menu.less';
```

Reply:
That's what I get for not setting up my precommit hooks!